ts/components/NewRemindEmailValidationOverlay.tsx

```diff
@@ -100,7 +101,18 @@ const NewRemindEmailValidationOverlay = (props: Props) => {
     [dispatch]
   );
 
-  // function to localize the title of the button. If the email is validated and if it is not, whether the confirmation email was sent or not
+  useOnFirstRender(() => {
+    // if the verification email was never sent, we send it
+    if (sendEmailAtFirstRender) {
+      sendEmailValidation();
+      IOToast.show(I18n.t("email.newvalidate.toast"));
```

Comment:
I like the use of the "Optimistic UI" approach. BTW, if we got an error, what happens to our countdown?

Reply:
i've fix this logic! Anyway thank you for highlighting this problem!